layouts/partials/head.html

```diff
@@ -21,31 +21,34 @@
 <!-- Icons -->
 <link rel="apple-touch-icon-precomposed" sizes="144x144" href="{{ "img/apple-touch-icon-144-precomposed.png" | absURL }}">
 {{ if isset $.Site.Params "favicon" }}
-<link rel="shortcut icon" href="{{ $.Site.Params.favicon | absURL }}">
+  <link rel="shortcut icon" href="{{ $.Site.Params.favicon | absURL }}">
 {{ else }}
-<link rel="shortcut icon" href="{{ printf "img/favicon/%s.png" $.Site.Params.ThemeColor | absURL }}">
+  <link rel="shortcut icon" href="{{ printf "img/favicon/%s.png" $.Site.Params.ThemeColor | absURL }}">
 {{ end }}
 
 <!-- Twitter Card -->
 <meta name="twitter:card" content="summary" />
-<meta name="twitter:title" content="{{ if .IsHome }}{{ $.Site.Title }} — {{ $.Site.Params.Subtitle }}{{ else }}{{ .Title }} :: {{ $.Site.Title }} — {{ $.Site.Params.Subtitle }}{{ end }}" />
-<meta name="twitter:description" content="{{ if .IsHome }}{{ $.Site.Params.tagline }}{{ else }}{{ .Summary | plainify }}{{ end }}" />
-<meta name="twitter:site" content="{{ $.Site.BaseURL }}" />
-<meta name="twitter:creator" content="{{ .Params.Author }}" />
-<meta name="twitter:image" content="{{ with .Params.Cover }}{{ . | absURL }}{{ end }}">
+{{ if (isset .Params "twitterSite") | or (eq .Params.twitterSite "") }}
+  <meta name="twitter:site" content="{{ $.Site.siteTwitter }}" />
+{{ end }}
+<meta name="twitter:creator" content="{{ if .IsHome }}{{ $.Site.twitterOwner }}{{ else }}{{ .Params.Author }}{{ end }}" />
```

Comment:
Should this param be `twitterCreator` instead of `twitterOwner`? The only reason I went with `twitterOwner` is because I felt this was more descriptive as to what the param is doing?

Reply:
Right! We should set `twitterCreator`.

Reply:
Done